cms/bundles/utils.py

```diff
@@ -22,3 +32,99 @@ def get_pages_in_active_bundles() -> list[int]:
     return list(
         bundle_page_class.objects.filter(parent__status__in=ACTIVE_BUNDLE_STATUSES).values_list("page", flat=True)
     )
+
+
+def _create_content_dict_for_pages(pages: list[tuple[dict[str, Any], str]]) -> list[dict[str, Any]]:
+    """Helper function to create content dictionary for article and methodology pages."""
+    article_pages: list[dict[str, Any]] = []
+    methodology_pages: list[dict[str, Any]] = []
+    content: list[dict[str, Any]] = []
+
+    for serialized_page, page_type in pages:
+        match page_type:
+            case "StatisticalArticlePage":
+                article_pages.append(serialized_page)
+            case "MethodologyPage":
+                methodology_pages.append(serialized_page)
+    if article_pages:
+        content.append({"type": "release_content", "value": {"title": "Publications", "links": article_pages}})
+    if methodology_pages:
+        content.append({"type": "release_content", "value": {"title": "Methodology", "links": methodology_pages}})
+    return content
+
+
+def serialize_page(page: "Page") -> dict[str, Any]:
+    """Serializes a page to a dictionary."""
+    return {
+        "id": uuid.uuid4(),
+        "type": "item",
+        "value": {"page": page.pk, "title": "", "description": "", "external_url": ""},
+    }
+
+
+def serialize_preview_page(page: "Page", bundle_id: int, is_previewable: bool) -> dict[str, Any]:
+    specific_page = page.specific_deferred
+    if workflow_state := specific_page.current_workflow_state:
+        state = workflow_state.current_task_state.task.name
+    else:
+        state = "not in a workflow"
```

Comment:
Let's update this to "Draft" here and on the bundle edit page to align with the language used by the publishing officers.